fiftyone/core/cli.py

```diff
@@ -3385,6 +3407,36 @@ def execute(parser, args):
         fo.pprint(op._doc)
 
 
+class DelegatedOutputCommand(Command):
+    """Prints the output for a delegated operation.
+
+    Examples::
+
+        # Print the output for a delegated operation
+        fiftyone delegated output <id>
+    """
+
+    @staticmethod
+    def setup(parser):
+        parser.add_argument("id", metavar="ID", help="the operation ID")
+
+    @staticmethod
+    def execute(parser, args):
+        dos = food.DelegatedOperationService()
+        op = dos.get(ObjectId(args.id))
+        if op.run_state == fooe.ExecutionRunState.COMPLETED:
+            result = op._doc.get("result", None)
+            if result:
+                result = result.get("result", None)
+            if result:
+                _print_dict_as_table(result)
```

Comment:
do we want to print something if there's no results, or just leave with no output?

Reply:
Personal preference I guess, but no output feels clean to me